lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -339,7 +339,7 @@ svi_autostate:
   kind: boolean
   get_value: '/^(?:no )?autostate$/'
   set_value: "<state> autostate"
-  default_value: true
+  default_value: false
```

Comment:
Reverting for now.  I only ran into trouble with this from the beaker tests, so I will revisit then.